site/documentation/content/api/tenant/index.md

```diff
@@ -176,7 +176,9 @@ If the *trusted-ca* property is provided, then it MUST contain a non-empty array
 | Name                     | Mandatory  | Type          | Default Value | Description |
 | :------------------------| :--------: | :------------ | :------------ | :---------- |
 | *subject-dn*             | *yes*      | *string*      |               | The subject DN of the trusted root certificate in the format defined by [RFC 2253](https://www.ietf.org/rfc/rfc2253.txt). Trusted CAs of the same tenant MAY share the same subject DN (e.g. allowing for the definition of overlapping validity periods). However, trusted CAs of different tenants MUST NOT share the same subject DN in order to allow for the look up of a tenant by the subject DN of one of its trusted CAs. |
-| *public-key*             | *yes*       | *string*     |               | The Base64 encoded binary DER encoding of the trusted root certificate's public key. |
+| *public-key*             | *yes*       | *string*     |               | The Base64 encoded binary DER encoding of the trusted root certificate's public key.  |
+| *not-before*             | *no*       | *string*     |               | The [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations) string representation of the start date of the trusted certificate's validity period.|
```

Comment:
I'd rather put the validity period props after the *algorithm* property so that the algorithm is *closer* the public key.
IMHO it would also be good to explain that the validity period does not need to be verified by the client because the payload will only contain CAs which are valid at the time of retrieval.

Reply:
ok, updated the pr